### ISSUE TITLE
Error 255: Phantomas error on rocssti.net ?

### ISSUE THREAD

**Issue by nico3333fr** [gmetais/YellowLabTools#114]
Hi,

I don't know why I get this error while testing http://rocssti.net/

Any idea ? Would it be an issue with some security headers ?
```
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-XSS-Protection: 1; mode=block
Content-Security-Policy: default-src 'self';  script-src 'self' www.google-analytics.com stats.g.doubleclick.net ; style-src 'self' data: ; img-src 'self' www.google-analytics.com stats.g.doubleclick.net data: ;  frame-src 'self' ; report-uri /csp-parser.php ;
```

Kind regards,
Nicolas

**Comment by gmetais** [comment]
Hi Nicolas,

Sorry I'm off in holidays, far from my computer, I won't be able to help you in the next 3 weeks :/

**Comment by nico3333fr** [comment]
Hi,

no problem, holidays are the most important thing :D 

Keep me informed when you will be able to, no worries ;)

**Comment by gmetais** [comment]
Hi Nicolas,

The problem is going to be hard to debug, as it works perfectly on my local instance of YLT, but refuses to work on the public instance.

I'm going to dig into it when I get more time.

PS: Did you notice that your beautiful large background images were loaded twice: 1 SVG + 1 PNG?

**Comment by nico3333fr** [comment]
Hi Gael,

no problem, I hope you will find the bug.

I've made a few tests (don't know if it can help you) : 
- removing X-Content-Type-Options: nosniff , X-Frame-Options: DENY , X-XSS-Protection: 1; mode=block and CSP header
- activating XHTML or simple HTML type MIME 

The error remains :-\

See you,
Nicolas

P.S : yes, I've built it in progressive enhancement using modernizr (for SVG), but you're right, I've prefered usng the SVG and provide a fallback :) (thanks for the remark)

**Comment by gmetais** [comment]
The run fails when I use the PhantomJS V2 engine and not with PhantomJS V1.9 engine. This is unusual.

Sorry, I'm not able to debug further...

**Comment by gmetais** [comment]
This issue refers to this bug: https://github.com/macbre/phantomas/issues/558

**Comment by nico3333fr** [comment]
Okay, hope they will find an issue :)

**Comment by gmetais** [comment]
Hi @nico3333fr 

I released a version 1.8.1 which resolves your bug. When a run fails with the webkit2 engine, it retries with webkit1.

**Comment by nico3333fr** [comment]
Hi @gmetais 

fine, it works now. Happy to see a 97/100 :)

I hope this annoyng webkit2 engine bug will be soon killed.

Thanks again,
Nicolas
